keras_nlp/models/roberta.py

```diff
@@ -30,32 +30,32 @@ class RobertaCustom(keras.Model):
 
     This network implements a bi-directional Transformer-based encoder as
     described in ["RoBERTa: A Robustly Optimized BERT Pretraining Approach"](https://arxiv.org/abs/1907.11692).
-    It includes the embedding lookups and transformer layers, but not include
-    the masked language model network.
+    It includes the embedding lookups and transformer layers, but not the masked
+    language model or classification task networks.
```

Comment:
Maybe keep the original, but say...

but does not include the masked language model head used during pretraining.

Unlike BERT there is no classification head used in pretraining, so the sentence you have is a bit confusing.